rosdep/base.yaml

```diff
@@ -2172,6 +2177,7 @@ liburdfdom-headers-dev:
     trusty: [liburdfdom-headers-dev]
     utopic: [liburdfdom-headers-dev]
     vivid: [liburdfdom-headers-dev]
+  gentoo: [media-libs/urdfdom_headers]
```

Comment:
These are from the Gentoo-ROS-Overlay I linked in the description. Versions 0.2.7, 0.3.0 and 0.3.0.